docs/samples/management-api-v2-walkthrough/5-catalog.md

```diff
@@ -0,0 +1,45 @@
+# Fetching provider's Catalog
+
+## Old plain JSON Schema
+
+```json
+{
+  "protocol" : "ids-protocol-http",
+  "providerUrl": "http://provider-control-plane:8282/api/v1/ids",
+  "querySpec": null
+}
+```
+
+## New JSON-LD Document
+
+> Please note: In our samples, properties **WILL NOT** be explicitly namespaced, and internal nodes **WILL NOT** be typed, relying on `@vocab` prefixing and root schema type inheritance respectively.
+
+```json
+{
+  "@context": {
+    "vocab": "https://w3id.org/edc/v0.0.1/ns/"
+  },
+  "protocol" : "dataspace-protocol-http",
+  "providerUrl": "http://provider-control-plane:8282/api/v1/dsp",
+  "querySpec": null
```

Comment:
what's the point of passing a null querySpec? it would be better to avoid it, and maybe adding another example with a query spec set.